src/lexer/regexp.ts

```diff
@@ -136,6 +142,8 @@ export function scanRegularExpression(parser: ParserState, context: Context): To
  */
 function validate(parser: ParserState, pattern: string, flags: string): RegExp | null | Token {
   try {
+    // Temporarily allows older version of Nodejs (or browser) to accept the new Indices flag "d".
+    flags = flags.replace('d', '');
```

Comment:
will it return a wrong value, even if the `/d` is supported?

Reply:
Pls elaborate.

Reply:
e.g. `value: /./d` =>  `/./` (as the `'/d'` has been replaced)?

Reply:
ok. seems it won't happen (as stated in the valid test). :)